concent_api/concent_api/settings/testing.py

```diff
@@ -60,14 +60,14 @@
 
 STORAGE_CLUSTER_ADDRESS = 'http://localhost/'
 
-CONCENT_MESSAGING_TIME    = 2
```

Comment:
Why are you removing this ?

Reply:
This should be still in our testing settings. It doesn't matter that we overriding settings in our tests (at least we should) but in `testing.py` we must have all settings what we have in `base.py`

Reply:
It was duplicated. look at line 69 - there is ` CONCENT_MESSAGING_TIME    = 0`

Reply:
So this should stay and delete this with value 0.